app/src/main/java/agency/five/codebase/android/movieapp/ui/component/ActorCard.kt

```diff
@@ -0,0 +1,95 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+
+import agency.five.codebase.android.movieapp.ui.theme.MovieAppTheme
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import coil.compose.rememberAsyncImagePainter
+import coil.compose.rememberImagePainter
+
+
+data class ActorCardViewState(
+    val imageURL: String,
+    val name: String,
+    val character: String,
+)
+
+@Composable
+fun ActorCard(
+    actorCardViewState: ActorCardViewState,
+    modifier: Modifier = Modifier,
+) {
+    Card(
+        Modifier
+            .size(width = 125.dp, height = 209.dp)
+            .padding(10.dp),
+        elevation = 10.dp
+    ) {
+        Box(modifier.fillMaxSize()) {
```

Comment:
I used a Box instead of column because I wanted to achieve a slightly different design. I used the box on top of the image to have the bottom part of the card with a gradient so the actor name and role is still visible on top of the picture. This is a slightly different design than in Figma. In the first homework pdf says that we can take the Figma design as a guidance and not stick to it fully. Is this ok then, or should I follow the Figma design fully for future homeworks?